### PR TITLE
Feature: upgrade balance endpoint

### PR DESCRIPTION
**Pull request by 1yam**
Problem:
  Currently, the Balance API Endpoint only returns the total balance for an address. And user (also frontend) as no easy way to get number of locked token.

On : 
`/api/v0/addresses/{address}/balance`

Solution:
  Enhance the Balance API Endpoint to include the "locked_amount" field in the response, indicating the amount of tokens in use for a specific address.

Example:

```json
{
    "address": "0x9319Ad3B7A8E0eE24f2E639c40D8eD124C5520Ba",
    "balance": 22192,
    "locked_amount": 2726
}

```

### CODE REVIEW

**Review by odesenfans — CHANGES_REQUESTED** [review]
Looks good, just add tests.

File: tests/conftest.py [hunk 1]
```diff
@@ -126,3 +150,179 @@ async def ccn_api_client(
...
+        address="0x9319Ad3B7A8E0eE24f2E639c40D8eD124C5520Ba",
+        chain=Chain.ETH,
+        balance=Decimal(22_192),
+        eth_height=0,
+    )
+
+    with session_factory() as session:
+        session.add(balance)
+        session.commit()
+        insert_volume_refs(session, fixture_instance_message)
```
Comment (odesenfans) [inline tests/conftest.py:326]: I don't understand why you insert the volumes in this fixture, as it has nothing to do with the balance. I expect 3 fixtures:

* `instance_message`
* `instance_message_with_volumes_in_db` (depends on the first one)
* `user_balance` (depends on nothing).

File: tests/api/test_balance.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    AlephBalanceDb,
+)
+MESSAGES_URI = "/api/v0/addresses/0x9319Ad3B7A8E0eE24f2E639c40D8eD124C5520Ba/balance"
+
+@pytest.mark.asyncio
+async def test_get_balance(ccn_api_client, user_balance: AlephBalanceDb):
+    response = await ccn_api_client.get(MESSAGES_URI)
+    assert response.status == 200, await response.text()
+    data = await response.json()
+    assert data["balance"] == 22192
```
Comment (odesenfans) [inline tests/api/test_balance.py:12]: Use the value from the fixture.
```suggestion
    assert data["balance"] == user_balance.balance
```

File: tests/api/test_balance.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    assert data["locked_amount"] == 6
```
Comment (odesenfans) [inline tests/api/test_balance.py:13]: How come this value is so low?

**Review by odesenfans — CHANGES_REQUESTED** [review]

File: tests/api/test_balance.py (hunk 3 above)
Comment (1yam) [reply tests/api/test_balance.py]: now it's implement too, just miss to add

**Review by odesenfans — APPROVED** [review]

**Pull request merged**